chapter05/Task_5_2.py

```diff
@@ -0,0 +1,117 @@
+                points = points - int(how_many)
+                break
+
+    elif choice == '3':
+        while points >= 0:
+            how_many = input("How many points do you want to bet health?\n")
+            if points - int(how_many) < 0 or points - int(how_many) > 30:
+                print("You can't do this. you don't have enough points")
+                print('You have left', points, 'points.')
+            elif int(specification['health']) + int(points) + int(how_many) < 0:
+                print('Enter only positive values')
+                print('You have left', points, 'points.')
+            else:
+                specification['health'] = int(specification['health']) + int(how_many)
+                points = points - int(how_many)
+                break
+
+    elif choice == '4':
+        while points >= 0:
+            how_many = input("How many points do you want to bet wisdom?\n")
+            if points - int(how_many) < 0 or points - int(how_many) > 30:
+                print("You can't do this. you don't have enough points")
+                print('You have left', points, 'points.')
+            elif int(specification['wisdom']) + int(points) + int(how_many) < 0:
+                print('Enter only positive values')
+                print('You have left', points, 'points.')
+            else:
+                specification['wisdom'] = int(specification['wisdom']) + int(how_many)
+                points = points - int(how_many)
+                break
+
+    elif choice == '5':
+        while points >= 0:
+            how_many = input("How many points do you want to bet agility?\n")
+            if points - int(how_many) < 0 or points - int(how_many) > 30:
+                print("You can't do this. you don't have enough points")
+                print('You have left', points, 'points.')
+            elif int(specification['agility']) + int(points) + int(how_many) < 0:
+                print('Enter only positive values')
+                print('You have left', points, 'points.')
+            else:
+                specification['agility'] = int(specification['agility']) + int(how_many)
+                points = points - int(how_many)
+                break
+
+    else:
+        print("Menu havan't this number")
+        continue
+
+    print('The hero looks like this:')
+    print("Strength - [{}], health - [{}], wisdom - [{}], agility - [{}]".format(specification['Strength'],
+                                                                                 specification['health'],
+                                                                                 specification['wisdom'],
+                                                                                 specification['agility']))
+    print('You have {} points left'.format(points))
+    print('\n')
+    choice = input('Press "0" for quiet from program or press "Enter" if you want see main menu.\n')
```

Comment:
'Press "0" **to quit the** program or press "Enter" if you want **to return to the main** menu.\n'